src/symbolize/symbolizer.rs

```diff
@@ -564,6 +567,7 @@ impl Symbolizer {
                 )
             },
         )?;
+        nsi.exit_mntns(ns_cookie)?;
```

Comment:
It feels like this "exit" should happen automatically as part of a destructor invocation via some guard. We are already leaking it on the error path above...

Reply:
Indeed, now I removed the `NsCookie` and call `setns`, `set_current_dir` in NsInfo's `drop`. 

BTW, in `drop` method, if `set_current_dir(oldcwd)` failed,  what is the impact on us? Should it panic or ignore it?

Reply:
Ignoring is reasonable and in-line with how [`std::fs::File`](https://doc.rust-lang.org/std/fs/struct.File.html) behaves, for example:

> Files are automatically closed when they go out of scope. Errors detected on closing are ignored by the implementation of Drop. Use the method [sync_all](https://doc.rust-lang.org/std/fs/struct.File.html#method.sync_all) if these errors must be manually handled.

That being said, let's log a `warn!` as I mentioned elsewhere so that there is at least some kind of trace of what happened.